modules/querying/pages/query-operations.adoc

```diff
@@ -596,6 +606,16 @@ a|`-SINGLE`
 |Enables a new internal framework for compiling non-distributed queries. Equivalent to the `single_gpr` session parameter described on the xref:ddl-and-loading:system-and-language-basics.adoc#_session_parameters[Session Parameters page.]
 
 The loop variable in a `xref:querying:control-flow-statements.adoc#_foreach_statement[FOREACH` loop] with this option enabled is treated as a copy, while normally, changes to the loop variable will change the value in the set or bag expression.
+
+a|`-COST`
+|Installs and optimizes the query using the query optimizer.
+This option automatically finds the most efficient traversal plan for executing the query and can reduce the time and memory it takes to execute a query.
+You can benefit from this option if you query uses a `FROM` clause that performs any number of hops.
```

Comment:
I do say that depending on the type of optimization used it could improve performance in different ways, so they can expect different kinds of optimizations in the future;

I'll change the verbiage to path patterns